README.md

```diff
@@ -1,109 +1,35 @@
-# Laravel example tutorial
+# Auth0 Laravel Samples
 
-This is a tutorial on how to run an example laravel application that uses auth0 for authentication, we have it in two flavor, as a local application using apache, or in the cloud using heroku
+These samples demonstrate how to add authentication to Laravel application with Auth0. Each folder contains a distinct application so that various Auth0 features can be viewed in isolation. You can read about these examples in our [Laravel Quickstart](https://auth0.com/docs/quickstart/webapp/laravel).
 
-## Clone the example
+## Embedded Integration Samples
```

Comment:
'Embedded Integration' is confusing, I don't think we use that term in any other readme. It can be confused with 'Embedded Login', which is not what we are doing here.

Reply:
@aaguiarz - Good call. Removed that whole section.